backend/src/v5/services/modelProcessing.js

```diff
@@ -98,6 +98,7 @@ const queueDrawingUpload = async (teamspace, project, model, revId, data, fileBu
 
 		publish(events.QUEUED_TASK_UPDATE, { teamspace, model, corId: revId, status: processStatuses.QUEUED });
 	} catch (err) {
+		console.log(err);
```

Comment:
@carmenfan do we want to keep this console log?

Reply:
@Charence  what console log? 🙈  (removed)